### PR TITLE
feat: allow multiple timers for multi-canvas rendering

### PR DESCRIPTION
**Pull request by rafrocha**
This PR allows multiple concurrent rendering by allowing different timers to be added whenever the WordCloud function is called. 
When I was trying to render wordcloud in multiple canvases in React - separate components - I noticed the single timer instance can end up stopping the wrong canvas from putting words.

### CODE REVIEW

File: src/wordcloud2.js [hunk 1]
```diff
@@ -81,7 +81,9 @@ if (!window.clearImmediate) {
     // "clearZeroTimeout" is implement on the previous block ||
     // fallback
     function clearImmediateFallback (timer) {
-      window.clearTimeout(timer)
+      for (const timerId in timer) {
+        window.clearTimeout(timer[timerId])
+      }
```
Comment (timdream) [inline src/wordcloud2.js]: Revert this change. The `timer` passed in is already the ID.

**Review by timdream — COMMENTED** [review]
Nice catch. This bug must been in the code for a decade already.

[`setImmediate`](https://developer.mozilla.org/en-US/docs/Web/API/Window/setImmediate) was a failed API from Microsoft. WordCloud is mostly likely ticking with `setZeroTimeout` implementation to queue the macro tasks on every browser. I wish I can written this part w/ rAF, which is more likely what everyone expects.

File: src/wordcloud2.js (hunk 1 above)
Comment (rafrocha) [reply src/wordcloud2.js]: Ah good catch, reverted.

**Comment by rafrocha** [comment]
Thank you for your timely response! Is there a new version with this change coming out soon?

Appreciate it, Tim! @timdream

**Comment by timdream** [comment]
***@***.***` published.
